CMakeLists.txt

```diff
@@ -4,6 +4,9 @@
 
 cmake_minimum_required(VERSION 3.3)
 
+# Setting -std=c++11
+set(CMAKE_CXX_STANDARD 11)
+
```

Comment:
Maybe also add `set(CXX_STANDARD_REQUIRED ON)` here since the build breaks on GCC without the C++11 standard?

Reply:
It sound like a good idea, I'll test it and if it don't break I will add it

Reply:
So it has broken anything on my computer, I have added it the changes to the branch